components/infobox/commons/infobox_person.lua

```diff
@@ -231,7 +231,10 @@ function Person:_setLpdbData(args, links, status, personType)
 		type = personType,
 		earnings = self.totalEarnings,
 		links = links,
-		extradata = {},
+		extradata = {
+			givenname = args.givenname,
+			familyname = args.familyname,
```

Comment:
I think it's better to call the fields `firstname` and `finalname`. Firstname would contain everything to display before the _id_, and finalname everything after. That way we only need to do the display mapping when it comes to certain countries in this module, and not in the consumers of the data.

Reply:
I think for the input on the wiki it would be a bit more clear to keep it to givenname/familyname to keep it clear what is intended for contributors. I think finalname would be a bit too vague with what is meant. (first/lastname is also a possibility, but I think first/last could potentially be a bit more problematic where contributors would simple just add the very last and first name of a full name.

Reply:
i think it was mostly about the key names in extradata

Reply:
Yes, would be for the LPDB database keys, not the wikicode parameter names